q2_dada2/tests/test_denoise.py

```diff
@@ -24,6 +24,10 @@ def _sort_seqs(seqs):
     return sorted(list(seqs), key=lambda x: x.metadata['id'])
 
 
+def _sort_table(table):
```

Comment:
That wasn't necessary before, and I'm not sure what/why the changes in this PR prompted this change - did you run into a problem with unstable data during these tests?

Reply:
Biom.table.concat sorts the opposite axis, see line 3391 here: https://github.com/biocore/biom-format/blob/master/biom/table.py, seems weird that it is not an option since it means there is not an equivalent to the additive identity of zero since even if there is nothing to concatenate you sort the opposite axis.

I chose to add the sorting prior to comparing:
1. I wanted to minimize changes in the _denoise code.
2. I did not want to modify the expected tables except adding a column for the new BLANK sample (to make it more clear)
3. This was easy to implement and I think clear what it is doing, maybe I should add a comment about why it became necessary?